readthedocs/builds/models.py

```diff
@@ -803,6 +803,31 @@ def get_full_url(self):
         )
         return full_url
 
+    def get_version_name(self):
+        if self.version:
+            return self.version.verbose_name
+        return self.version_name
+
+    def get_version_slug(self):
+        if self.version:
+            return self.version.verbose_name
+        return self.version_name
+
+    def get_version_type(self):
+        if self.version:
+            return self.version.type
+        return self.version_type
```

Comment:
I wonder if we should just always use the denormalized values here. I don't think they'll ever change, hopefully.

Reply:
yeah, I think is better to use the saved values, since they are from when the build happened, if they change, they aren't part of that build anymore. I'm doing this for now till we fully migrate the data from the old builds. I'm worried that would take some time as we would be iterating over each build D: maybe do the data migration in another PR?